DfE.FindInformationAcademiesTrusts/Pages/privacy.cshtml

```diff
@@ -0,0 +1,152 @@
+@page
+
+@{
+  Layout = "_ContentLayout";
+  ViewData["Title"] = "Privacy notice";
+}
+
+<div class="govuk-width-container">
+  <main class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">
+          Privacy notice for Find information about academies and trusts
+        </h1>
+        <p class="govuk-body">
+          The Department for Education (DfE) owns ‘Find information about academies and trusts’ and is also the data controller for the personal data processed as part of the product.
+        </p>
+        <p class="govuk-body">
+          This notice explains why we use personal information and how.
+        </p>
+        <h3 class="govuk-heading-m">
+          What data is collected
+        </h3>
+        <p class="govuk-body">
+          We may use the following personal information:
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>trust and school managers and governance roles name and email addresses</li>
+          <li>authorised and public user’s internet protocol address (IP)</li>
+          <li>DfE SFSO lead and trust relationship manager names and email addresses</li>
+        </ul>
+        <h3 class="govuk-heading-m">
+          Why we use it
+        </h3>
+        <p class="govuk-body">
+          DfE processes data to compile and maintain a register of academies and trusts in England and Wales. This product is a tool for everyone in DfE to find information which is collated from multiple sources about academies and trusts to inform their assessments, decisions and support activities.
+        </p>
+        <h3 class="govuk-heading-m">
```

Comment:
I believe all of the h3 tags on this page need to be h2 tags, as there is no h2 on the page (should not skip heading levels)